forge-gui/res/cardsfolder/upcoming/portal_to_phyrexia.txt

```diff
@@ -0,0 +1,10 @@
+Name:Portal to Phyrexia
+ManaCost:9
+Types:Artifact
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigSac | TriggerDescription$ When CARDNAME enters the battlefield, each opponent sacrifices three creatures.
+SVar:TrigSac:DB$ Sacrifice | Defined$ Player.Opponent | SacValid$ Creature | Amount$ 3
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigChange | TriggerDescription$ At the beginning of your upkeep, put target creature card from a graveyard onto the battlefield under your control. It's a Phyrexian in addition to its other types.
+SVar:TrigChange:DB$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | ValidTgts$ Creature | GainControl$ True
```

Comment:
AnimateSubAbility$ missing

Reply:
You didn't use AnimateSubAbility$

Reply:
Hmm what's the difference between normal subability ?

Reply:
It needs to ETB as phyrexian:

> 611.2e If a resolving spell or ability both puts a nontoken permanent onto the battlefield and creates
a continuous effect stating that the permanent “is [characteristic],” that continuous effect applies
simultaneously with the permanent entering the battlefield. This characteristic is usually a color
or a creature type. If the continuous effect says the permanent “becomes [characteristic]” or
“gains [an ability],” that effect applies after the permanent is on the battlefield.

You need to use `Defined$ Remembered`

Reply:
Don't use RememberChanged & Cleanup, that's handled by the engine in this special case
Always check how other cards use it

Reply:
The RememberChanged$ is still there... xD

Reply:
Oh I see, build in with AnimateSubAbility ...